docker/diem-build.sh

```diff
@@ -60,7 +60,7 @@ if [ "$1" = "--incremental" ]; then
     echo "${BLUE}Tip:${RESTORE}: If you see unexpected failure, try docker rm -f $CONTAINER"
   fi
   CONTAINER_TARGET=/target
-  docker exec -e "STRIP_DIR=$CONTAINER_TARGET" -e "ENABLE_FAILPOINTS=$ENABLE_FAILPOINTS" -i -t "$CONTAINER" ./docker/build-common.sh --target-dir "$CONTAINER_TARGET"
+  docker exec -e "STRIP_DIR=$CONTAINER_TARGET" -e "ENABLE_FAILPOINTS=$ENABLE_FAILPOINTS" -i -t "$CONTAINER" "./docker/build-common.sh" --target-dir "$CONTAINER_TARGET"
```

Comment:
May I ask what is the difference with the quotation here?

Reply:
none, initially wanted to pass parameters to the script.

Reply:
but the script passes along all parameters to the rust build.

Reply:
gotcha